app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,38 @@
+class Api::SessionsController < ApplicationController
+  def create
+    user = User.find_by(email: session_params[:email])
+
+    if user && user.valid_password?(params[:password])
+      sign_in user
+      token = JWT.encode({ user_id: user.id }, ENV['DEVISE_JWT_SECRET_KEY'], 'HS256')
+      render json: { token: token }, status: :ok
+    else
+      render json: { error: 'Invalid email or password' }, status: :unauthorized
+    end
+  end
+
+  def new
+    user = User.find_by(email: params[:email])
+
+    if user && user.valid_password?(params[:password])
+      sign_in(user)
+      render json: { message: 'Đăng nhập thành công', user: user }
+    else
+      render json: { error: t("login.valid_email_password_combination") }, status: :unauthorized
+    end
```

Comment:
remove this
this method only render UI for login form 
Form login will move to folder devise/sessions/new.html.erb